Sources/Unbox.swift

```diff
@@ -482,6 +482,20 @@ public class Unboxer {
     
     /// Unbox a required Array containing values of a raw type
     public func unbox<T: UnboxableRawType>(key: String, isKeyPath: Bool = true) -> [T] {
+        if let rawValue = UnboxValueResolver<[T]>(self).resolveOptionalValueForKey(key, isKeyPath: isKeyPath) {
+            return rawValue
+        }
+        
+        let transformedStringArray = UnboxValueResolver<[String]>(self).resolveOptionalValueForKey(key, isKeyPath: isKeyPath, transform: { stringArray -> [T]? in
+            return stringArray.flatMap {
```

Comment:
Question is, should we really `flatMap` here? Since if the array contains an invalid element (Like a string that can't be converted to the required type), that should probably be treated as a data inconsistency error.